deploy/domain/datasource.tf

```diff
@@ -0,0 +1,11 @@
+variable "env" {}
+variable "route53_zone_name" {}
+
+data "aws_route53_zone" "zone" {
+  name         = var.route53_zone_name
+  private_zone = false
+}
+
+provider "aws" {
+  region     = "us-east-1"
```

Comment:
The region is repeated a few times in different TF files, and it seems important "for getting certificate", so I would put it in a variable (expression the certificate thingy by the way)

Reply:
The reason behind that is certificates must be created in us-east-1 IF you use a edge optimized gateway domain name. Therefore I think there is no real benefit to setting this as a variable, but maybe let a comment linking to a reference on this topic?